dev/docker-compose.ingress.yaml

```diff
@@ -0,0 +1,19 @@
+---
+version: "3.4"
+services:
+  fhir:
+    # expose HAPI to internet
+    labels:
+      - "traefik.enable=true"
+      - "traefik.http.routers.fhir-${COMPOSE_PROJECT_NAME}.rule=Host(`fhir.${BASE_DOMAIN:-localtest.me}`)"
+      - "traefik.http.routers.fhir-${COMPOSE_PROJECT_NAME}.entrypoints=websecure"
+      - "traefik.http.routers.fhir-${COMPOSE_PROJECT_NAME}.tls=true"
+      - "traefik.http.routers.fhir-${COMPOSE_PROJECT_NAME}.tls.certresolver=letsencrypt"
+    networks:
+      - ingress
+
+networks:
+  # ingress network
+  ingress:
+    external: true
+    name: external_web
```

Comment:
This section _shouldn't_ be necessary, since it's already in the ["base" file were extending from](https://github.com/uwcirg/dcw-environments/blob/main/dev/docker-compose.yaml#L211-L214)
```suggestion
```